### ISSUE TITLE
JSON schema enums not resolved via $ref in the schema 

### ISSUE THREAD

**Issue by josdejong** [josdejong/jsoneditor#302]
The editor doesn't create a dropdown for enums in a JSON schema which have to be resolved via a reference `$ref` like in:

```
{
    "type": "array",
    "items": { "$ref": "#/definitions/myEnum" },
    "definitions": {
        "myEnum": {
            "enum": ["one", "two", "three"]
        }
    }
}`
```

See also https://github.com/josdejong/jsoneditor/pull/285#issuecomment-221183267

**Comment by maaikez** [comment]
Thanks for adding this issue. What is the priority of this feature?

**Comment by josdejong** [comment]
It's not a bug so there is no extremely high priority on this. If you find time to implement this before I do, feel free to contribute :).

**Comment by maaikez** [comment]
That was indeed what I thought. There is no time for the project I used the json editor for anymore but if I can do it at home, I will try to help (but that's not very soon).

**Comment by zgz682000** [comment]
but why "$ref" works on https://www.jsoneditoronline.org/

**Comment by josdejong** [comment]
The JSON schema error messages work but there is no enum dropdown created

**Comment by josdejong** [comment]
Fixed via #423
